packages/eslint-config-angular/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
I'll comment here since it's the first in the file view. All the public packages should behave as if they are standalone and as such include README, LICENSE, and their own standalone configs, ie .eslintrc.js configs. There shouldn't be coupling to other packages without going through standard npm dependency resolution.